### PR TITLE
Added fwartner/mac-cleanup

### PR DESCRIPTION
**Pull request by fwartner**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mac-cleanup.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class MacCleanup < Formula
+  desc "Cleanup script for OSX"
+  homepage "https://github.com/fwartner/mac-cleanup/"
+  url "https://github.com/fwartner/mac-cleanup/archive/1.0.2.tar.gz"
+  sha256 "52c454c9abac16801814a90d766ff1d7367a698b0e003bc3dfe02cdfde6d1fa6"
+
+  def install
+    chmod 0755, "cleanup.sh"
```
Comment (SMillerDev) [inline Formula/mac-cleanup.rb:8]: Why is this needed?

File: Formula/mac-cleanup.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class MacCleanup < Formula
+  desc "Cleanup script for OSX"
+  homepage "https://github.com/fwartner/mac-cleanup/"
+  url "https://github.com/fwartner/mac-cleanup/archive/1.0.2.tar.gz"
+  sha256 "52c454c9abac16801814a90d766ff1d7367a698b0e003bc3dfe02cdfde6d1fa6"
+
```
Comment (SMillerDev) [inline Formula/mac-cleanup.rb:6]: needs a `depends_on :macos`

File: Formula/mac-cleanup.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  def install
+    chmod 0755, "cleanup.sh"
+    bin.install "cleanup.sh" => "cleanup"
+  end
+
+  test do
+    system "#{bin}/cleanup"
```
Comment (SMillerDev) [inline Formula/mac-cleanup.rb:13]: This should check if the command had any effect.

**Comment by SMillerDev** [comment]
Closing this as it's not carrying an open source license. See https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

**Comment by HenrikBacher** [comment]
@SMillerDev project has been updated with the MIT License: https://github.com/fwartner/mac-cleanup/blob/master/LICENSE
Can this PR be re-opened?

**Comment by SMillerDev** [comment]
It's better to just make a new pull request.
